doc/src/guides/contributing/dev-setup.rst

```diff
@@ -49,10 +49,10 @@ After the configuration, your setup should be similar to this:
 .. code-block:: bash
 
    $ git remote -v
-   origin	https://github.com/<your-user-name>/sympy (fetch)
-   origin	https://github.com/<your-user-name>/sympy (push)
-   upstream	https://github.com/sympy/sympy (fetch)
-   upstream	https://github.com/sympy/sympy (push)
+   origin   https://github.com/<your-user-name>/sympy (fetch)
+   origin   https://github.com/<your-user-name>/sympy (push)
```

Comment:
single space

Reply:
I aligned the URLs to match visual expectation: In reality `git remote -v` do use a tab here:

```bash
$ git remote -v | cat -A
mdk^Igit@github.com:JulienPalard/sympy.git (fetch)$
mdk^Igit@github.com:JulienPalard/sympy.git (push)$
origin^Ihttps://github.com/sympy/sympy.git (fetch)$
origin^Ihttps://github.com/sympy/sympy.git (push)$
```

so in a terminal it'll (often) be aligned, like:

```bash
$ git remote -v
mdk	    git@github.com:JulienPalard/sympy.git (fetch)
mdk     git@github.com:JulienPalard/sympy.git (push)
origin  https://github.com/sympy/sympy.git (fetch)
origin  https://github.com/sympy/sympy.git (push)
```

but not always:

```bash
$ git remote rename mdkmdkm  mdkmdkmd
$ git remote -v
mdkmdkmd        git@github.com:JulienPalard/sympy.git (fetch)
mdkmdkmd        git@github.com:JulienPalard/sympy.git (push)
origin  https://github.com/sympy/sympy.git (fetch)
origin  https://github.com/sympy/sympy.git (push)
```

(Yeah, on github I had to manually replace tabs with spaces to make it look like in my terminal, as github and my terminal don't interpret tabs the same way...)

Reply:
Sorry, I probably didn't understand what I was looking at. Just thought it was manually type code and figured it should be consistent.